protostar/commands/test/expected_event.py

```diff
@@ -40,38 +42,45 @@ def __init__(
     def __str__(self) -> str:
         result: List[str] = []
 
-        result.append(f'"name": "{self.name}"')
+        result.append(f'"name": ({self.name})"{get_selector_from_name(self.name)}"')
```

Comment:
The string representation of `ExpectedEvent` should match it's interface. The user may try to use a "selector" instead of a "name" in the `expect_events` cheatcode to fix the error. I'd create a new virtual field — "selector" just to not confuse the user.